java/flight/flight-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcDriver.java

```diff
@@ -93,14 +93,7 @@ protected String getFactoryClassName(final JdbcVersion jdbcVersion) {
 
   @Override
   protected DriverVersion createDriverVersion() {
-
-    CreateVersionIfNull:
-    {
-
-      if (version != null) {
-        break CreateVersionIfNull;
-      }
-
+    if (version != null) {
       try (Reader reader = new BufferedReader(new InputStreamReader(
           this.getClass().getResourceAsStream("/properties/flight.properties"),
```

Comment:
getResourceAsStream returns null if the path isn't found. Suggest we throw something more helpful if we get that (this will probably be an NPE)

Reply:
Done.